src/runner/index.js

```diff
@@ -126,6 +137,10 @@ export default class Runner extends EventEmitter {
                 return this._runTask(reporterPlugin, browserSet, tests);
             });
 
-        return Runner._createCancelablePromise(runTaskPromise);
+        return this._createCancelablePromise(runTaskPromise);
+    }
+
+    async stop () {
+        await Promise.all(this.pendingTaskPromises.slice().map(taskPromise => taskPromise.cancel()));
```

Comment:
We need a comment about `slice` as well. This is not quite ovbious thing, so we need to describe it (see BrowserSet)